<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_Scene_ManualCapture</class>
 <widget class="QWidget" name="Dlg_Scene_ManualCapture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dlg_Scene_ManualCapture</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="Dlg_PictureDraw" name="picDraw" native="true">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>600</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>600</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_loadPicScene">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>加载图片场景</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_illSnap">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>违停抓拍</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Dlg_PictureDraw</class>
   <extends>QWidget</extends>
   <header>dlg_picturedraw.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
